src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="body">
+    <article class="card" data-qa="card">
+      <div class="card__image-container">
+        <img
+          src="images/imac.jpeg"
+          class="card__image"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+      </div>
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="card__details">
```

Comment:
Try to make class name as descriptive as possible. I would name it 
```suggestion
      <div class="card__review-wrapper">
```